dotnet/src/Functions/Functions.OpenAPI/Model/RestApiOperation.cs

```diff
@@ -59,6 +59,11 @@ public sealed class RestApiOperation
     /// </summary>
     public IList<RestApiOperationParameter> Parameters { get; }
 
+    /// <summary>
+    /// The list of possible operation responses.
+    /// </summary>
+    public IDictionary<string, RestApiOperationResponse> Responses { get; }
```

Comment:
ResponseSchemas?

Reply:
For now I've named the new class `RestApiOperationExpectedResponse`. Would appreciate feedback on this naming. I like `Responses` still just because that matches the OpenAPI definitions but could see renaming to `ExpectedResponses`. Thoughts?

Reply:
Resolving as-is for now.